fhir-examples/src/main/resources/json/ibm/basic/BasicQuantity.json

```
@@ -82,6 +82,15 @@
           "code": "s"
         }
       }
+    }, 
+    {
+        "url": "http://example.org/Quantity-WithExponent",
```

*super* minor, but I've been using lowercase after the `-`
```suggestion
        "url": "http://example.org/Quantity-withExponent",
```